stl/inc/execution

```diff
@@ -1216,6 +1245,8 @@ void for_each(_ExPo&&, _FwdIt _First, _FwdIt _Last, _Fn _Func) noexcept /* termi
             }
         }
 
+        _For_each_ivdep(_UFirst, _ULast, _Pass_fn(_Func));
+    } else if constexpr (remove_reference_t<_ExPo>::_Parallelize) {

```

Comment:
This exactly repeats line 1230 - I think you intended to test `_Ivdep` here.

Reply:
Pro: My brain sees these sorts of things.
Con: My brain sees spaces and commas everywhere.